src/ripple/app/main/Application.cpp

```diff
@@ -501,7 +501,7 @@ class ApplicationImp
             logs_->journal("ValidatorList"), config_->VALIDATION_QUORUM))
 
         , validatorSites_ (std::make_unique<ValidatorSite> (
-            get_io_service (), *validators_, logs_->journal("ValidatorSite")))
+            get_io_service (), *validators_, logs_->journal("ValidatorSite"), *config_))
```

Comment:
Radical concept... change the ctor to only accept an application instance reference, and store it rather than the other references. Yes, it could potentially limit the configuration of the object in some future use case, but not at this moment. Besides, the complexity can be added on an as needed basis.

Reply:
done